iphone/Classes/TiUIiOSDocumentViewerProxy.m

```diff
@@ -25,6 +25,7 @@ - (UIDocumentInteractionController *)controller
   if (controller == nil) {
     NSURL *url = [self _toURL:[self valueForUndefinedKey:@"url"] proxy:self];
     controller = [[UIDocumentInteractionController interactionControllerWithURL:url] retain];
+    controller.name = [self valueForUndefinedKey:@"title"];
```

Comment:
`controller.name` will be `nil` if the `title` property is not passed, is it fine as well?

Reply:
Yes, that is the expected behavior, as it fallbacks to the default.

Reply:
I can't see the option to mark this conversation resolved myself. Seems I'm missing some permissions.